.github/workflows/build.yml

```diff
@@ -75,19 +55,16 @@ jobs:
 
       - name: Install TeXLive
         run: |
-          sudo apt update
+          sudo apt-get update
           sudo apt-get install texlive texlive-latex-extra
 
       - name: Install local pgip R package
         run: |
-          R -e "library(devtools); devtools::install_local('src/R/pgip')"
+          R -e "library(devtools); devtools::install_local('src/R/pgip', upgrade=F)"
 
       - name: Setup Quarto
         uses: quarto-dev/quarto-actions/setup@v2
 
-      - name: Install bcftools
-        run: ./scripts/install-bcftools.sh
-
```

Comment:
This however is trickier. It's been awhile since I checked, but the latest version of conda bcftools had dependency issues that interfered with - oh, I don't remember now - but the end result was I *had* to install manually. I'll see if things have changed now.